content/docs/command-reference/metrics/index.md

```diff
@@ -27,21 +21,76 @@ positional arguments:
 In order to track metrics associated to machine learning experiments, DVC has
 the ability to mark a certain stage <abbr>outputs</abbr> as files containing
 metrics to track (see the `--metrics` option of `dvc run`). Metrics are
-project-specific floating-point values e.g. `AUC`, `ROC`, etc.
-
-Supported file formats: JSON. Metrics can be organized in a tree hierarchy in a
-JSON file. DVC addresses the metrics by the tree path.
+project-specific floating-point or integer values e.g. `AUC`, `ROC`,
+`False-Positives` etc.
+
+[Show](/doc/command-reference/metrics/show), and
+[diff](/doc/command-reference/metrics/diff), commands are available to manage
+<abbr>DVC project</abbr> metrics.
+
+## Formats and metrics types
```

Comment:
```suggestion
### Formats and types of metrics
```
(H3 if we agree this is part of the description so no need for extra entry in right sidebar)